.github/workflows/main.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         id: cache
         with:
           path: .venv
-          key: venv-${{ runner.os }}-${{ steps.full-python-version.outputs.version }}-${{ hashFiles('**/poetry.lock') }}
+          key: venv-1-${{ runner.os }}-${{ steps.full-python-version.outputs.version }}-${{ hashFiles('**/poetry.lock') }}
```

Comment:
Is there a reason for this?

Reply:
This was just to force a fresh install without a cache hit, before the renamed module was not being found. See https://github.com/python-poetry/poetry-plugin-bundle/runs/6097368079?check_suite_focus=true:

```
ModuleNotFoundError: No module named 'poetry_bundle_plugin'
```